src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/HintRow.jsx

```diff
@@ -1,48 +1,43 @@
 import React from 'react';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import {
-  Col, Container, Form, Icon, IconButton, Row,
+  ActionRow, Form, Icon, IconButton,
 } from '@edx/paragon';
-import { Delete } from '@edx/paragon/icons';
+import { DeleteOutline } from '@edx/paragon/icons';
 import PropTypes from 'prop-types';
 import messages from '../messages';
 
 export const HintRow = ({
   value,
   handleChange,
   handleDelete,
-  // inject
+  handleEmptyHint,
+  // injected
   intl,
 }) => (
-  <Container fluid>
-    <Row>
-      <Col xs={10}>
-        <Form.Group>
-          <Form.Control
-            value={value}
-            onChange={handleChange}
-            floatingLabel={intl.formatMessage(messages.hintInputLabel)}
-          />
-        </Form.Group>
-      </Col>
-
-      <Col xs={2}>
-        <IconButton
-          src={Delete}
-          iconAs={Icon}
-          alt={intl.formatMessage(messages.settingsDeleteIconAltText)}
-          onClick={handleDelete}
-          variant="secondary"
```

Comment:
are we sure we don't want `variant="secondary"` in the new version?

Reply:
yes